### DISCUSSION TITLE
How to change the background color when testing something white?

### DISCUSSION THREAD

**Discussion by Ricardo1980** [pointfreeco/swift-snapshot-testing#563] [Q&A]
I am testing a SwiftUI button that has a white background.
Because the background is already white, it is a bit difficult to see is the component is working fine.
Is there any way to change the background color?
Or should I manually embed my component into another view with a different background color?
Thanks for suggestions.

**Comment by mbrandonw** [comment]
Using a non-white background view to wrap your button is probably the way to go.

I'm going to convert this to a discussion since it is not an issue with the library.

**Comment by CraigSiemens** [comment]
We've run a similar issue were we wanted a transparent background in the image, similar to how the UIView(Controller) snapshots are created. Currently the only workaround we see is duplicating [SwiftUIView.swift](https://github.com/pointfreeco/swift-snapshot-testing/blob/main/Sources/SnapshotTesting/Snapshotting/SwiftUIView.swift) to be able to set the background of the UIHostingController to clear.
